tree/ntuple/v7/src/RFieldSequenceContainer.cxx

```diff
@@ -586,6 +586,7 @@ void ROOT::Experimental::RVectorField::RVectorDeleter::operator()(void *objPtr,
 {
    auto vecPtr = static_cast<std::vector<char> *>(objPtr);
    if (fItemDeleter) {
+      R__ASSERT(fItemSize > 0);
```

Comment:
This looks unrelated to the emulation itself

Reply:
I found a crash while testing it and it didn't look like it was worth a separate PR (or even commit)...